README.md

```diff
@@ -42,49 +42,53 @@ in sync with the written specification on conventionalcommits.org.
 
 ```ebnf
 /* See: https://tools.ietf.org/html/rfc3629#section-4 */
-<UTF8-char>     ::= "Placeholder for UTF-8 grammar"
-<UTF8-octets>   ::= <UTF8char>+
-
-<CR>            ::= "0x000D"
-<LF>            ::= "0x000A"
-<newline>       ::= [<CR>], <LF>
-<parens>        ::= "(" | ")"
-<ZWNBSP>        ::= "U+FEFF"
-<TAB>           ::= "U+0009"
-<VT>            ::= "U+000B"
-<FF>            ::= "U+000C"
-<SP>            ::= "U+0020"
-<NBSP>          ::= "U+00A0"
+<UTF8-char>       ::= "Placeholder for UTF-8 grammar"
+<UTF8-octets>     ::= <UTF8char>+
+
+<CR>              ::= "0x000D"
+<LF>              ::= "0x000A"
+<newline>         ::= [<CR>], <LF>
+<parens>          ::= "(" | ")"
+<ZWNBSP>          ::= "U+FEFF"
+<TAB>             ::= "U+0009"
+<VT>              ::= "U+000B"
+<FF>              ::= "U+000C"
+<SP>              ::= "U+0020"
+<NBSP>            ::= "U+00A0"
 /* See: https://www.ecma-international.org/ecma-262/11.0/index.html#sec-white-space */
-<USP>           ::= "Any other Unicode 'Space_Separator' code point"
+<USP>             ::= "Any other Unicode 'Space_Separator' code point"
 /* Any non-newline whitespace: */
-<whitespace>    ::= <ZWNBSP> | <TAB> | <VT> | <FF> | <SP> | <NBSP> | <USP>
-
-<message>       ::= <summary>, <newline>+, <body>, <newline>*, <footer>+
-                 |  <summary>, <newline>*, <footer>+
-                 |  <summary>, <newline>*
-
-<summary>       ::= <type>, "(", <scope>, ")", ["!"], ":", <whitespace>*, <text>
-                 |  <type>, ["!"], ":", <whitespace>*, <text>
-<type>          ::= <any UTF8-octets except newline or parens or ":" or "!:" or whitespace>+
-<scope>         ::= <any UTF8-octets except newline or parens>+
-<text>          ::= <any UTF8-octets except newline>*
-
-/*
- * Note: if the first <body> node starts with "BREAKING CHANGE:" this should
- * be treated by parsers as a breaking change marker upstream:
- */
-<body>          ::= [<any text except pre-footer>], <newline>, <body>*
-                 | [<any text except pre-footer>]
-/* Note: <pre-footer> is used during parsing, but never returned in the AST. */
-<pre-footer>    ::= <newline>*, <footer>+
-
-<footer>        ::= <token>, <separator>, <whitespace>*, <value>, [<newline>]
-<token>         ::= "BREAKING CHANGE"
-                 |  <type>, "(" <scope> ")", ["!"]
-                 |  <type>, ["!"]
-<separator>     ::= ":" | " #"
-<value>         ::= <text>, <continuation>+
-                 | <text>
-<continuation> ::= <newline>, <whitespace>+, <text>
+<whitespace>      ::= <ZWNBSP> | <TAB> | <VT> | <FF> | <SP> | <NBSP> | <USP>
+
+<message>         ::= <summary>, <newline>+, <body>, <newline>*, <footer>+
+                   |  <summary>, <newline>*, <footer>+
+                   |  <summary>, <newline>*
+
+/* "!" should be added to the AST as a <breaking-change> node with the value "!" */
```

Comment:
Added note that `!` should be added as a `breaking-change` node to the AST (_this is already the behavior_).